File: 프로그래머스/0/120895. 인덱스 바꾸기/인덱스 바꾸기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string my_string, int num1, int num2) {
    string answer = "";
    char temp = '0';
    for(int i = 0; i < my_string.size(); i++) {
        if (i == num1) {
            temp = my_string[i];
            my_string[i] = my_string[num2];
            my_string[num2] = temp;
            }
    }
    answer = my_string;
    
    return answer;
}

File: 프로그래머스/0/120895. 인덱스 바꾸기/인덱스 바꾸기.py
def solution(my_string, num1, num2):
    answer = ''
    my_list = list(my_string)
    for i in range(len(my_list)):
        if i == num1:
            temp = my_list[i];
            my_list[i] = my_list[num2]
            my_list[num2] = temp
    answer = ''.join(my_list)
    return answer